### PR TITLE
♻️Refactor BPMN-Studio CSS

### LINKED ISSUE #288: ♻️Refactor CSS Files
#### Describe your issue

Our CSS files are chaos.

#### How can others reproduce the issue?

Look at the app.scss.

#### Possible solution

Have a look at this: https://github.com/process-engine-pro/design_mockup_bpmn-studio/blob/master/styles.css

Let's adapt to this style.

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [x] I've checked if this issue already exists
- [x] I've included all the information that i think is relevant
- [x] I've added logs and/or screenshots (if applicable)
- [x] I've mentioned PRs and issues that relate to this one

### PR DESCRIPTION
**Pull request by alexanderkasten**
## What did you change?

This PR refactors the css part of the bpmn-studio modules.
Only the order of the css rules has been changed, so it should has no effect.

Fixes #288 

## How can others test the changes?

- checkout the branch
- `npm run build`
- `npm run electron-start-dev`

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

File: src/app.scss [hunk 1]
```diff
@@ -17,10 +17,10 @@ a:hover {
 
 .settings-glyphicon {
   top: 12px;
+  margin-right: 5px;
+  padding: 0 !important;
```
Comment (SteffenKn) [inline src/app.scss]:
```
padding: 0 !important;
margin-right: 5px;
```

File: src/modules/bpmn-diff-view/bpmn-diff-view.scss [hunk 2]
```diff
@@ -21,9 +21,9 @@
 
 .lower-diff-view {
   position: relative;
+  display: inline-flex;  
   height: 67%;
   width: 100%;
```
Comment (SteffenKn) [inline src/modules/bpmn-diff-view/bpmn-diff-view.scss]:
```
width: 100%;
height: 67%;
```

File: src/modules/bpmn-io/bpmn-io.scss [hunk 3]
```diff
@@ -37,9 +37,9 @@
 
 .resize-div {
   position: relative;
+  left: -3px;
   width: 6px;
   height: 100%;
-  left: -3px;
   float:left;
```
Comment (SteffenKn) [inline src/modules/bpmn-io/bpmn-io.scss]:
````
  position: relative;
  left: -3px;
  float: left;
  width: 6px;
  height: 100%;
```

File: src/modules/bpmn-xml-view/bpmn-xml-view.scss [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
   height: 100%;
-  bottom: 0;
+  background-color: unset;
   border: none;
   word-break: unset;
   word-wrap: unset;
-  background-color: unset;
+  line-height: 13px;
+  font-size: 13px;
+  font-family: monospace;
```
Comment (SteffenKn) [inline src/modules/bpmn-xml-view/bpmn-xml-view.scss]:
```
  font-size: 13px;
  font-family: monospace;
  line-height: 13px;
  word-break: unset;
  word-wrap: unset;
```

File: src/modules/diagram-tools-right/diagram-tools-right.scss [hunk 5]
```diff
@@ -11,12 +11,12 @@
 
 .tool {
   display: block;
-  text-align: center;
-  padding: 5px;
   margin: 0 5px;
-  border: none;
+  padding: 5px;
```
Comment (SteffenKn) [inline src/modules/diagram-tools-right/diagram-tools-right.scss]:
```
  padding: 5px;
  margin: 0 5px;
```

File: src/modules/diagram-tools-right/diagram-tools-right.scss [hunk 6]
```diff
@@ -60,28 +60,28 @@
...
   margin: 0;
   overflow: hidden;
-  cursor: pointer;
-  display: inline-block;
-  border: 0;
   background: 0;
+  border: 0;
   color: #333;
   vertical-align: middle;
+  cursor: pointer;
```
Comment (SteffenKn) [inline src/modules/diagram-tools-right/diagram-tools-right.scss]:
```
  display: inline-block;
  margin: 0;
  overflow: hidden;
  cursor: pointer;
  background: 0;
  border: 0;
  color: #333;
  vertical-align: middle;
```

File: src/modules/navbar/navbar.scss [hunk 7]
```diff
@@ -70,8 +70,8 @@ button {
 }
 
 .menu-tabbed-link:hover {
-  text-decoration: none;
   color: #5e5e5e;
+  text-decoration: none;
   cursor: default;
```
Comment (SteffenKn) [inline src/modules/navbar/navbar.scss]:
```
  cursor: default;
  color: #5e5e5e;
  text-decoration: none;
```

File: src/modules/navbar/navbar.scss [hunk 8]
```diff
@@ -85,25 +85,25 @@ button {
...
+  top: 40px;
+  padding: 0;
   background-color: #f7f7f7;
   border-top: none;
-  top: 40px;
   border-top-left-radius: 0;
   border-top-right-radius: 0;
   box-shadow: none;
-  padding: 0;
   cursor: pointer;
```
Comment (SteffenKn) [inline src/modules/navbar/navbar.scss]:
```
  top: 40px;
  padding: 0;
  cursor: pointer;
  background-color: #f7f7f7;
  border-top: none;
  border-top-left-radius: 0;
  border-top-right-radius: 0;
  box-shadow: none;
```

File: src/modules/process-solution-panel/process-solution-panel.scss [hunk 9]
```diff
@@ -1,8 +1,8 @@
 .process-explorer__solution-explorer {
-  background-color: #f7f7f7;
   width: 220px;
-  border-right: 2px solid #dcdbdb;
   height: 100%;
+  background-color: #f7f7f7;
+  border-right: 2px solid #dcdbdb;
   overflow-y: auto;
   overflow-x: hidden;
```
Comment (SteffenKn) [inline src/modules/process-solution-panel/process-solution-panel.scss]:
```  
  width: 220px;  
  height: 100%;
  overflow-y: auto;
  overflow-x: hidden;
  background-color: #f7f7f7;
  border-right: 2px solid #dcdbdb;
  user-select: none;
```

File: src/modules/process-solution-panel/process-solution-panel.scss [hunk 10]
```diff
@@ -15,8 +15,8 @@
 
 .index-card {
   padding: 8px 12px;
-  cursor: pointer;
   color: darkgray;
+  cursor: pointer;
```
Comment (SteffenKn) [inline src/modules/process-solution-panel/process-solution-panel.scss]: The order was right.
```
  padding: 8px 12px;
  cursor: pointer;
  color: darkgray;
```

File: src/modules/process-solution-panel/process-solution-panel.scss [hunk 11]
```diff
@@ -70,9 +70,9 @@
 .process-explorer__structure-item {
   padding: 2px 10px;
   overflow: hidden;
-  cursor: pointer;
   text-overflow: ellipsis;
   white-space: nowrap;
+  cursor: pointer;
```
Comment (SteffenKn) [inline src/modules/process-solution-panel/process-solution-panel.scss]: The order was right.
```
  padding: 2px 10px;
  overflow: hidden;
  cursor: pointer;
  text-overflow: ellipsis;
  white-space: nowrap;
```

File: src/modules/process-solution-panel/process-solution-panel.scss [hunk 12]
```diff
@@ -98,8 +98,8 @@
 }
 
 .solution-group__close:hover {
-  cursor: pointer;
   color: #5e5e5e;
+  cursor: pointer;
```
Comment (SteffenKn) [inline src/modules/process-solution-panel/process-solution-panel.scss]: The order was right.
```
  cursor: pointer;
  color: #5e5e5e;
```

File: src/modules/process-solution-panel/process-solution-panel.scss [hunk 13]
```diff
@@ -115,8 +115,8 @@
 }
 
 .solution-diagram-group__link:hover {
-  cursor: pointer;
```
Comment (SteffenKn) [inline src/modules/process-solution-panel/process-solution-panel.scss]: The order was right.
```
  cursor: pointer;
  text-decoration: underline;
```

File: src/modules/process-solution-panel/process-solution-panel.scss [hunk 14]
```diff
@@ -132,8 +132,8 @@
 }
 
 .single-diagram-group__link:hover {
-  cursor: pointer;
```
Comment (SteffenKn) [inline src/modules/process-solution-panel/process-solution-panel.scss]: The order was right.
```
  cursor: pointer;
  text-decoration: underline;
```

File: src/modules/status-bar/status-bar.scss [hunk 15]
```diff
@@ -6,18 +6,18 @@
...
   background-color: #9c27b0;
   background-color: #607d8b;
+  border-top: 2px solid #e9e9e9;
   color: #fff;
+  flex-direction: row;
+  justify-content: space-evenly;
+  align-items: stretch;
   font-size: 11px;
   user-select: none;
   cursor: default;
```
Comment (SteffenKn) [inline src/modules/status-bar/status-bar.scss]:
```
  position: fixed;
  bottom: 0;
  display: flex;
  width: 100%;
  height: 22px;
  padding: 0 5px;
  cursor: default;
  background-color: #3f51b5;
  background-color: #9c27b0;
  background-color: #607d8b;
  border-top: 2px solid #e9e9e9;
  user-select: none;
  color: #fff;
  font-size: 11px;
  flex-direction: row;
  justify-content: space-evenly;
  align-items: stretch;
```

**Comment by Paulomart** [comment]
Can we _please_ add linting for CSS files?

**Pull request merged**